SwahiliBox/swahilibox-events-api
# Setup basic app

#### What does this PR do?
sets up a basic application and server
Also sets up `eslint` 

#### How should this be manually tested?
Pull these changes, run `yarn install` then `yarn start` or `yarn start:dev`
Then head over to the index route. `GET`  `localhost:5000` 
You should get a welcome message.
